### PR TITLE
Analyze without dev deps

### PR DESCRIPTION
**Pull request by szepeviktor**
This should reveal the problem behind #1

### CODE REVIEW

File: .github/workflows/phpstan.yml [hunk 1]
```diff
@@ -20,9 +20,12 @@ jobs:
...
+          tools: phpstan
 
       - name: Install composer dependencies
         uses: ramsey/composer-install@v3
+        with:
+          composer-options: "--no-dev"
 
       - name: Run PHPStan
-        run: ./vendor/bin/phpstan --error-format=github
+        run: phpstan
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:34]: Error format is automatic inside GHA.

**Comment by szepeviktor** [comment]
No CI for PRs :(

**Comment by szepeviktor** [comment]
This won't work. Composer cannot ignore dev deps.
laravel/framework says it replaces all illuminate packages.

e.g. https://github.com/composer/composer/issues/6011

**Comment by szepeviktor** [comment]
Removing `require-dev` in the workflow.

**Comment by xHeaven** [comment]
Looks good to me, thanks for your contribution again!

> No CI for PRs :(

I'll fix that a bit later, not going to lie, I wasn't expecting contributions 👀

**Pull request merged**